gateway-ha/src/test/java/io/trino/gateway/ha/TestGatewayHaSingleBackend.java

```diff
@@ -36,9 +36,9 @@
 import static org.testcontainers.utility.MountableFile.forClasspathResource;
 
 @TestInstance(Lifecycle.PER_CLASS)
-final class TestGatewayHaSingleBackend
+class TestGatewayHaSingleBackend
```

Comment:
We should add an abstract test class. Otherwise, we can't run specific tests with `TestEnvConfiguration` easily on IDE.